Formula/docker-machine.rb

```diff
@@ -1,11 +1,11 @@
 class DockerMachine < Formula
   desc "Create Docker hosts locally and on cloud providers"
   homepage "https://docs.docker.com/machine"
-  url "https://github.com/docker/machine.git",
-      tag:      "v0.16.2",
-      revision: "bd45ab13d88c32a3dd701485983354514abc41fa"
+  url "https://gitlab.com/gitlab-org/ci-cd/docker-machine.git",
+      tag:      "v0.16.2-gitlab.18",
+      revision: "cd8285a7e2310276c7d20575f15bba40a0678ed9"
```

Comment:
```suggestion
  url "https://gitlab.com/gitlab-org/ci-cd/docker-machine.git",
      tag:      "v0.16.2-gitlab.18",
      revision: "cd8285a7e2310276c7d20575f15bba40a0678ed9"
  version "0.16.2-gitlab.18"
```
Or something like that. `brew` is parsing the version as `18`.